kpops/cli/exception.py

```diff
@@ -1,2 +1,2 @@
 class ClassNotFoundError(Exception):
-    """Similar to builtin ModuleNotFoundError; class doesn't exist inside module."""
```

Comment:
How did you choose where to use single or double back tick?

Reply:
that's one of the things I am quite unsure about, I usually just reused what was there already. but noticed that there's both ways throughout the codebase. do you know if there's a difference?

Reply:
[There is](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup), but I am not sure if it matter at all in our case

Reply:
I checked the highlighting in nvim but for me there's no difference between single and double backtick